corehq/apps/analytics/tests/test_partner_analytics_utils.py

```diff
@@ -127,7 +120,7 @@ def delete_user(user):
                 pass
         for user in cls.users:
             elastic_user = transform_user_for_elasticsearch(user.to_json())
-            send_to_elasticsearch('users', elastic_user)
+            user_adapter.index(elastic_user, refresh=True)
```

Comment:
simply amazing. 😻

Reply:
this comment is not about the specific line...just about how clean all of this looks

Reply:
this comment is not about the specific line...just about how clean all of this looks

Reply:
Yep. All credits to @joxl for laying out the foundation 🌟